docs/admin/authentication.md

```diff
@@ -286,7 +286,7 @@ To enable the plugin, configure the following flags on the API server:
 
 | Parameter | Description | Example | Required |
 | --------- | ----------- | ------- | ------- |
-| `--oidc-issuer-url` | URL of the provider which allows the API server to discover public signing keys. Only URLs which use the `https://` scheme are accepted.  This is typically the provider's discovery URL without a path, for example "https://accounts.google.com" or "https://login.salesforce.com".  This URL should point to the level below .well-known/openid-configuration | If the discovery URL is https://accounts.google.com/.well-known/openid-configuration the value should be https://accounts.google.com | Yes |
+| `--oidc-issuer-url` | URL of the provider which allows the API server to discover public signing keys. Only URLs which use the `https://` scheme are accepted.  This is typically the provider's discovery URL without a path, for example "[https://accounts.google.com](https://accounts.google.com)" or "[https://login.salesforce.com](https://login.salesforce.com)".  This URL should point to the level below .well-known/openid-configuration. | If the discovery URL is [https://accounts.google.com/.well-known/openid-configuration](https://accounts.google.com/.well-known/openid-configuration), the value should be [https://accounts.google.com](https://accounts.google.com). | Yes |
```

Comment:
The original formatting without links is correct -- the URLs should be rendered on the page as strings (values to be provided), not as links. Please revert this change.

Reply:
Thanks. But when you edit origina page, then preview it,  thoes all are links.